CHANGELOG.md

```diff
@@ -1,4 +1,319 @@
+        * In extensions/v1beta1 if CPUUtilization in HorizontalPodAutoscalerSpec was not
+specified it was set to 80 by default while in autoscaling/v1 HPA object
+without TargetCPUUtilizationPercentage specified is a valid object. Pod
+autoscaler controller will apply a default scaling policy in this case which is
+equivalent to the previous one but may change in the future.
+     * The previous version, <code>apiVersion: extensions/v1beta1</code>, is still supported.  Even if you roll back to 1.1, the objects created using
+the new apiVersions will still be accessible, using the old version.  You can
+continue to use your existing JSON and YAML files until you are ready to switch
+to <code>autoscaling/v1</code>.  We may remove support for HorizontalPodAutoscalers with  <code>apiVersion: extensions/v1beta1 </code>in 1.3 or 1.4.
+  * Kube-Proxy now defaults to an iptables-based proxy. If the --proxy-mode flag is
+specified while starting kube-proxy (‘userspace’ or ‘iptables’), the flag value
+will be respected. If the flag value is not specified, the kube-proxy respects
+the Node object annotation: ‘net.beta.kubernetes.io/proxy-mode’. If the
+annotation is not specified, then ‘iptables’ mode is the default. If kube-proxy
+is unable to start in iptables mode because system requirements are not met
+(kernel or iptables versions are insufficient), the kube-proxy will fall-back
+to userspace mode. Kube-proxy is much more performant and less
+resource-intensive in ‘iptables’ mode. 
+  * Node stability can be improved by reserving [resources](https://github.com/kubernetes/kubernetes/blob/release-1.2/docs/proposals/node-allocatable.md) for the base operating system using --system-reserved and --kube-reserved Kubelet flags
+  * Liveness and readiness probes now support more configuration parameters:
+periodSeconds, successThreshold, failureThreshold
+  * The new ReplicaSet API (Beta) in the Extensions API group is similar to
+ReplicationController, but its [selector](http://kubernetes.io/docs/user-guide/labels/#label-selectors) is more general (supports set-based selector; whereas ReplicationController
+only supports equality-based selector).
+  * Scale subresource support is now expanded to ReplicaSets along with
+ReplicationControllers and Deployments. Scale now supports two different types
+of selectors to accommodate both [equality-based selectors](http://kubernetes.io/docs/user-guide/labels/#equality-based-requirement) supported by ReplicationControllers and [set-based selectors](http://kubernetes.io/docs/user-guide/labels/#set-based-requirement) supported by Deployments and ReplicaSets.
+  * “kubectl run” now produces Deployments (instead of ReplicationControllers) and
+Jobs (instead of Pods) by default.
+  * Pods can now consume Secret data in environment variables and inject those
+environment variables into a container’s command-line args.
+  * Stable version of Heapster which scales up to 1000 nodes: more metrics, reduced
```

Comment:
I guess this item includes what I mentioned in node team release notes:

New Alpha API in Kubelet - `/stats/summary`  in a user friendly format with reduced system overhead for both Kubelet and Heapster

This is great, but I still think we should keep one separately as an experimental feature since we do have users directly scraping kubelet for metrics, and not use heapster. I want those users switch or prepare to switch to the new API exposed by Kubelet.